packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/SelectedPersona.tsx

```diff
@@ -114,7 +114,7 @@ const SelectedPersonaInner = React.memo(
         dragDropSubscription?.dispose();
         setDragDropSubscription(undefined);
       };
-    }, [dragDropHelper, dragDropSubscription, root, events, dragDropOptions]);
+    }, [dragDropHelper]);
```

Comment:
Can you run lint? This is actually what I had in the first place, but lint didn't like it. If not, is there some way to disable it? (Or rewrite this?)